CIP-????/spec.cddl

```diff
@@ -0,0 +1,22 @@
+; CIP-XXXX: CIP-Descriptor
+; Version: 1
+
+; Define any common or reusable pieces such as strings, URIs, etc
+
+string = text .size (0..64)
+
+; Define any nested objects before the top-level object
+
+royalty-object = {
+  0: string       ; rate
```

Comment:
should this string become integer, now that we're using a 6-decimal-shifted integer for the rate key tags?

```suggestion
  0: integer       ; rate
```

Reply:
Yes, that could be correct if switching to the offset integer math for royalty fee rates